tools/nebula-spark/pom.xml

```diff
@@ -21,17 +21,12 @@
     <dependencies>
         <dependency>
             <groupId>org.apache.spark</groupId>
-            <artifactId>spark-core_2.12</artifactId>
+            <artifactId>spark-core_2.11</artifactId>
             <version>${spark.version}</version>
         </dependency>
         <dependency>
             <groupId>org.apache.spark</groupId>
-            <artifactId>spark-graphx_2.12</artifactId>
```

Comment:
`graphx` will support later so don't remove it

Reply:
graphx is not removed,  I modified its version to 2.11.